app/code/Magento/Quote/Test/Unit/Model/CustomerManagementTest.php

```diff
@@ -147,7 +147,7 @@ protected function setUp(): void
 
     public function testPopulateCustomerInfo()
     {
-        $this->quoteMock->expects($this->once())
+        $this->quoteMock->expects($this->exactly(2))
```

Comment:
Is there any reasons to check how exactly times it was ran? AFAIK method getCustomer caches result, but it will not cause performance issues in case if it will be ran more than 1 or 2 times, but in future this test will fail if amount of running this method will change. 
Maybe we can replace it to `$this->atLeastOnce()`?